spec/features/create_object_h2_spec.rb

```diff
@@ -68,7 +68,7 @@
     expect(page).to have_content(item_title)
 
     # This happens asynchronously, it might take a bit
-    expect(page).to have_content('https://sul-purl-stage')
+    expect(page).to have_content('https://purl-stage.stanford.edu')
```

Comment:
this does not work in the qa environment.

Reply:
Sounds like we need this as a new settings variable so we can have env-specific variables.

Reply:
I think we can configure h2-qa because I think they're both pointing at the same purl (but different c-names)

Reply:
Done (https://github.com/sul-dlss/shared_configs/commit/707455d321af3f38c960cc3bcf47685dc4470fc5) and deployed.